src/components/VueTablePage.vue

```diff
@@ -28,25 +27,26 @@
           </div>
         </el-row>
       </div>
-      <div class="table-page-body">
+      <div
+        class="table-page-body"
+        :style="{ height: height}">
```

Comment:
Can't we use the shorthand syntax in this case? Like `{ height }`. 
It also looks like your linter failed here, otherwise it would have detected the missing whitespace before the closing curly bracket.